Cython/Utility/ExtensionTypes.c

```diff
@@ -131,16 +131,16 @@ static int __Pyx_PyType_Ready(PyTypeObject *t) {
             PyGC_Enable();
     #else
         if (gc_was_enabled) {
-            PyObject *t, *v, *tb;
-            PyErr_Fetch(&t, &v, &tb);
+            PyObject *ty, *v, *tb;
+            PyErr_Fetch(&ty, &v, &tb);
```

Comment:
I think other places use `tp` for a type reference. Don't know of any that use `ty`.
```suggestion
            PyObject *tp, *v, *tb;
            PyErr_Fetch(&tp, &v, &tb);
```